engine/dlib/src/dlib/profile/profile_remotery.cpp

```diff
@@ -24,9 +24,11 @@
 
 #include "dmsdk/external/remotery/Remotery.h"
 
-
 rmtU32 rmt_HashString32(const char* s, int len, rmtU32 seed)
 {
+    static uint32_t empty_hash = dmHashString32("<empty>");
+    if (len == 0)
+        return empty_hash;
```

Comment:
Although not the root cause, it seems like an ok check to make, since this api it public and also it's not included in shipped games.